### PR TITLE
Create delet.py

### PR DESCRIPTION
**Pull request by EdwardDowling**

### CODE REVIEW

File: steely/plugins/delet.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+'''
+
+__author__ = 'sam'
+COMMAND= '.delet'
+
+import subprocess
+
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    subprocess.Popen("rm -rf ../../steely".split())
```
Comment (iandioch) [inline steely/plugins/delet.py]: this will not work on Windows, is therefore insubmissable to the perfect steely codebase.

**Comment by byxor** [comment]
:thinking:

File: steely/plugins/delet.py (hunk 1 above)
Comment (sentriz) [reply steely/plugins/delet.py]: `os.rmdir()` maybe

**Comment by oskarmcd** [comment]
no unit tests == no submission, sorry sam

**Comment by byxor** [comment]
Remember to write an integration test too and run it before committing.

**Comment by sentriz** [comment]
how about a `shutil.rmtree(os.path.join(os.getcwd(), ".."))`

**Comment by sentriz** [comment]
oops sorry, tested it and `shutil.rmtree("../../", ignore_errors=True)` works

**Comment by byxor** [comment]
I vote to close this. It messes with .steelygh.

**Pull request closed without merging**